listenbrainz_spark/recommendations/tests/test_dataframe.py

```diff
@@ -0,0 +1,155 @@
+import uuid
+import unittest
+from datetime import datetime
+
+import listenbrainz_spark
+from listenbrainz_spark.tests import SparkTestCase
+from listenbrainz_spark.recommendations import create_dataframes
+from listenbrainz_spark import schema, utils, config, path, hdfs_connection, stats
+
+from pyspark.sql import Row
+
+# path used in between test functions of this class
+LISTENS_PATH = '/test/listens/' # HDFS path to listens
+MAPPING_PATH = '/test/mapping.parquet'
+MAPPED_LISTENS_PATH = '/test/mapped_listens.parquet'
+
+class CreateDataframeTestCase(SparkTestCase):
+
+    date = datetime.utcnow()
```

Comment:
I'd have preferred to put it as a member variable (self.date) instead of a class variable (cls.date) because it should be changed for every instance of the class but hey seems very nitpicky. Feel free to change it if you want.

Reply:
I am really up for the suggestion but I think that __init__ of this class will override __init__ of unittest.TestCase. Is there any other way to put it as a member variable other than initializing inside every method?

Reply:
Put it in setUp? You can define member variables anywhere, not just init.